Online-Messaging-App/src/app/home/sidebar/sidebar.component.scss

```diff
@@ -52,6 +52,9 @@ mat-sidenav-container {
     background-color: #282828;
     height: 100%;
 }
+.scrollable{
```

Comment:
![image](https://user-images.githubusercontent.com/35238045/73327510-1d304e80-420b-11ea-88a8-a4c1b5543a12.png)
Is there not a way to only make this appear when the list actually needs to be scrolled?

Reply:
like the scroll bar I mean

Reply:
Changed up the code but not sure if it works as I can't recreate issue on my end